web-frontend/package.json

```diff
@@ -20,11 +20,15 @@
     "vue-writer": "^2.0.2"
   },
   "devDependencies": {
-    "@vitejs/plugin-vue": "^3.2.0",
+    "@vitejs/plugin-vue": "^4.6.2",
     "@vue/test-utils": "^2.4.5",
     "happy-dom": "^14.7.1",
     "http-server": "^14.1.1",
-    "vite": "^3.2.10",
+    "vite": "^4.5.3",
     "vitest": "^1.6.0"
+  },
+  "optionalDependencies": {
```

Comment:
Using latest one for avoiding build error with `rollup`:
https://www.npmjs.com/package/@rollup/rollup-linux-arm64-gnu?activeTab=versions
https://www.npmjs.com/package/@rollup/rollup-linux-x64-gnu?activeTab=versions